typescript/README.md

```diff
@@ -0,0 +1,9 @@
+name: TypeScript
+
+description: A typed superset of JavaScript that compiles to plain JavaScript.
```

Comment:
Maybe we should use something a bit less technical.

One idea:

```suggestion
description: JavaScript + types = TypeScript
```

Another one:

```suggestion
description: JavaScript with types
```